webapp/src/components/common/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import Select, {MultiValue, SingleValue} from 'react-select';
+
+import Control from 'components/common/dropdown/control';
+
+export type DropdownOption = {
+    value: string,
+    label: string,
+};
+
+export type Props = {
+    options: Array<DropdownOption>
+    value?: DropdownOption
+    defaultValue?: DropdownOption
+    onChange: (newValue: DropdownOption) => void
+}
+const Dropdown = ({options, value, defaultValue, onChange}: Props) => {
+    const customComponents = useMemo(() => {
+        return {
+            Control,
+        };
+    }, []);
```

Comment:
This can be simplified to something like:

`const customComponents = useMemo(() => ({Control}), []);`

Reply:
Done 👍